### ISSUE TITLE
StytchError missing fields

### ISSUE THREAD

**Issue by markedwards** [stytchauth/stytch-node#314]
When testing the M2M flow using the B2BClient and inducing a 400 error by passing an invalid `client_id` or `client_secret`, the caught `StytchError` does not have all of the expected fields.

Example:
```typescript
try {
  const response = await stytchClient.m2m.token(data);
} catch (e) {
  if (e instanceof StytchError) {
    console.log(e.error_type); // <- undefined
    console.log(e.error_message);  // <- undefined
    console.log(e.error_url);  // <- undefined
  }
  raise e;
}
```

If I look at `e.message`, I see the expected values, but the keys are `"error"`, `"error_description"`, and `"error_uri"`, so I'm guessing there's a naming mismatch bug here.

**Comment by logan-stytch** [comment]
Hey @markedwards, thanks for the bug report! I've just landed #315 which will fix this. It has just gone out in 10.15.1 which is available now.

**Comment by markedwards** [comment]
Wow, thanks for the quick response!  I haven't even committed my diff yet!

Confirmed the fix on my end.
